bootstrap/storageclass/helmfile/values/nfs-client-provisioner.yaml.gotmpl

```diff
@@ -9,4 +9,4 @@ serviceAccount:
   name: nfs-client-provisioner
 nfs:
   path: "/nfs"
-  #server: Will be set outside
+  server: {{ .Values.global.nfs_server_ip }}
```

Comment:
Thins should probably be moved to its own topic in the config. So something like values.storageClass.serverIp. This to support other specific config in the storage class aswell. Im not sure about the logic of how to init / validate it though when having different settings depending on what storageClass we are using. Maybe look at the objectStorage solution for insperation

Reply:
> Thins should probably be moved to its own topic in the config. So something like values.storageClass.serverIp. This to support other specific config in the storage class aswell. Im not sure about the logic of how to init / validate it though when having different settings depending on what storageClass we are using. Maybe look at the objectStorage solution for insperation

Wouldn't it confuse users if we the same value at different places, specially if it is set manually?